core/templates/core/user_accounts/register/register.html

```diff
@@ -25,6 +25,7 @@ <h1>Register</h1>
             {% endif %}
 
             {% generic_field form.email -1 %}
+            {% generic_field form.username -1 %}
```

Comment:
waarom is username != nickname?

Reply:
een nickname kan spaties ed. bevatten; een username niet. Vandaar dat die split is gemaakt.